packages/generator-ds/README.md

```diff
@@ -3,9 +3,11 @@
 ## Getting Started
 
 1. Install [Yeoman](https://yeoman.io/) and this generator: `npm i -g yo @canonical/generator-ds`
-2. Run the generator: `yo @canonical/ds`
-3. Select a subgenerator from the list and follow its instructions
+2. Run the generator: `yo @canonical/ds:component`
```

Comment:
This isn't technically correct for the root generator readme - someone should just invoke `yo @canonical/ds` which will prompt them to select a specific generator such as `:component`.

That prompt doesn't appear currently as we only have one generator. I guess it's a question of simplicity. Should we document the recommended use to be `:component`  for now as we only have one generator, and change it back to `yo @canonical/ds` when there are more generators?